### DISCUSSION TITLE
Hw can I avoid a sub module update on each check out and pull

### DISCUSSION THREAD

**Discussion by kevinwhine** [gitextensions/gitextensions#8284] [General] [answered]
Hi,

I get a separate activity when I do a checkout of a branch in the local repository or do a "pull rebase" of the following:

"git" submodule update --init --recursive

The sub-modules does not move mostly so I did like to avoid the init and update activity which takes relatively a long time compared to the desired action.

How can I avoid the sub module init and update?

Thank you for your attention

**Comment by pmiossec** [comment, marked as answer]
Uncheck this setting:
![image](https://user-images.githubusercontent.com/460196/86457051-55403e00-bd23-11ea-9e8b-67b83e8e9d06.png)

**Comment by kevinwhine** [comment]
Thank you Philippe!

That is great.

Kevin

________________________________
